documentation/reindexing.html

```diff
@@ -61,3 +65,92 @@ <h3 id="reindexing-progress">Reindexing progress</h3>
 the cluster is ready, while <em>running</em> means it's currently progressing. Finally,
 <em>successful</em> means all documents of that type were successfully reindexed.
 </p>
+
+
+
+<h2 id="use-cases">Use cases</h2>
+<p>
+  Below are sample changes to the schema for different use cases,
+  or examples of operational steps for data manipulation.
+</p>
+<table class="table">
+  <tr id="clear-field">
+    <th >clear field</th>
+    <td>
+      <p>
+        To clear a field's data:
+<pre>
+# before
+field artist type string {
+    indexing:      summary | index
+}
+
+# after
+field artist type string {
+    indexing: "" | summary | index
```

Comment:
Can you do that to document fields, though?

Reply:
It's also potentially much more effective to feed that particular partial update to all documents in the corpus.

Reply:
This is not supported for document fields.

Reply:
correct, it will throw a deployment error